NumberValidator/Validators/IValidator.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace NumberValidator.Validators
 {
-    public interface IValidator
+    public interface IAadhaarValidator
```

Comment:
Why are you renaming the interface? Do you not know what an interface is for?

Reply:
its corrected